app/bounty_requests/templates/bounty_request_form.html

```diff
@@ -27,24 +27,53 @@
       {% include 'shared/nav.html' %}
     </div>
     <div class="row no-gutter">
-      <div class="col-12 body pb-1">
-        <div class="m-4">
-          <div>
-            <div class="request-form" id="primary_form">
-              <h3>{% trans "Request a Bounty" %} <span id="alpha">{% trans "Alpha" %}</span></h3>
-              <p>
-                {% blocktrans %}
-                Have an open-source issue that you think would benefit the community?
-                Suggest it be given a bounty!
-                {% endblocktrans %}
-              </p>
-              <div class="mb-3">
-                <label for="requested_by" class="form__label">{% trans 'Your Github Profile' %}
-                  {% if not user.is_authenticated %}
-                  (<a href="{% url 'social:begin' 'github' %}?next={{ request.get_full_path }}">{% trans 'Login' %}</a>)
-                  {% endif %}
-                </label>
-                <input name="requested_by" placeholder="{% trans "Please Login First" %}" id="requested_by" class="form__input" type="text" value="{% if github_handle %}{{ github_handle }}{% endif %}" required disabled />
+      <div class="col body">
+        <div class="container-fluid no-gutter">
+          <div class="row">
+            <div class="col-12 offset-md-1 col-md-10">
+              <div class="m-4">
+                <div>
+                  <div class="request-form" id="primary_form">
```

Comment:
this seems unrelated?

Reply:
Rebasing with upstream master, introduced this

Reply:
@iamonuwa could redo this PR to ensure only your changes are present ?